domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/EmptyFileErrorNotice.java

```diff
@@ -23,7 +23,7 @@
 
 public class EmptyFileErrorNotice extends ErrorNotice {
     public EmptyFileErrorNotice(String filename) {
-        super(filename, E_039,
+        super(filename, E_047,
```

Comment:
It appeared that this error code E039 was not the error code to be used for `EmptyFileErrorNotice` to ensure consistency with `RULES.md` doc. This probably comes an error when merging https://github.com/MobilityData/gtfs-validator/pull/317. 

Should I rather open a new issue referencing this, and apply this change in a new PR, so that this PR does not contain changes unrelated to  https://github.com/MobilityData/gtfs-validator/issues/162? @barbeau

Reply:
Yes, I'd reference this in a new issue and separate PR, to avoid mixing unrelated things there.

Reply:
See https://github.com/MobilityData/gtfs-validator/issues/345